sentry/src/transport.rs

```diff
@@ -222,9 +222,12 @@ implement_http_transport! {
                         }
                     }
 
+                    let mut body = vec![];
```

Comment:
Out of curiosity, I tend to write this as `Vec::new()` but you use the macro, are there any advantages to using the macro?  (my choice is purely because I like code with fewer macros)